src/AR/ARModel.tsx

```diff
@@ -1,16 +1,24 @@
 import React from 'react';
 import { Viro3DObject } from '@reactvision/react-viro';
 
-interface ModelProps {
+interface ARModelProps {
+  key: number;
   url: string;
   position: [number, number, number];
   rotation: [number, number, number];
   scale: [number, number, number];
 }
 
-const ARModel: React.FC<ModelProps> = ({ url, position, rotation, scale }) => {
+const ARModel: React.FC<ARModelProps> = ({
+  key,
+  url,
+  position,
+  rotation,
+  scale,
+}) => {
   return (
     <Viro3DObject
+      key={key}
```

Comment:
What was the reason for adding the key prop? Is this error/warning valid risk?
![image](https://github.com/user-attachments/assets/662a58e8-958b-4b3f-8c14-9328cf0eda33)

Reply:
I actually checked it and it isn't needed so I removed it 👍